applications/convection_diffusion_application/python_scripts/convection_diffusion_base_solver.py

```diff
@@ -228,12 +229,22 @@ def AddVariables(self):
         
         self.print_on_rank_zero("::[ConvectionDiffusionBaseSolver]:: ", "Variables ADDED")
 
+
     def GetMinimumBufferSize(self):
         return 2
 
     def AddDofs(self):
         # this can safely be called also for restarts, it is internally checked if the dofs exist already
         settings = self.main_model_part.ProcessInfo[KratosMultiphysics.CONVECTION_DIFFUSION_SETTINGS]
+        print("***********************************")
```

Comment:
Please use Logger, and this is not a very clean solution :(

Reply:
to be removed